content/en/references/coverage/index.md

```diff
@@ -113,20 +113,20 @@ The table is a work in progress and will be updated as more functions are implem
 |BUILD_SCOPED_FILE_URL|❓|❓|
 |BUILD_STAGE_FILE_URL|❓|❓|
 |CASE|❓|❓|
-|CAST , ::|❓|❓|
+|CAST|❓|❓|
 |CBRT|❓|❓|
 |CEIL|❓|❓|
 |CHARINDEX|❓|❓|
 |CHECK_JSON|❓|❓|
 |CHECK_XML|✅|✅|
-|CHR , CHAR|❓|❓|
+|CHR, CHAR|❓|❓|
 |COALESCE|✅|✅|
 |COLLATE|❓|❓|
 |COLLATION|❓|❓|
 |COMPLETE (SNOWFLAKE.CORTEX)|❓|❓|
 |COMPLETE_TASK_GRAPHS|❓|❓|
 |COMPRESS|❓|❓|
-|CONCAT , |||✅|✅|
+|CONCAT|✅|✅|
```

Comment:
Same here I would think `, ||` is to suggest `||` as an alternative syntax for `CONCAT`:
https://docs.snowflake.com/en/sql-reference/functions/concat